### DISCUSSION TITLE
Simple TUI mode (like nvidia-smi, rocm-smi)

### DISCUSSION THREAD

**Discussion by Umio-Yasuno** [Umio-Yasuno/amdgpu_top#14] [Ideas]
## Purpose
- Monitor multiple AMD GPUs on a single screen
- Limit the information to be displayed

![](https://user-images.githubusercontent.com/53935716/237839143-a28b84af-c3e1-45c4-b1cc-1472a0ba8ac9.png)

**Comment by Umio-Yasuno** [comment]
### Update (2023-05-20)
![smi](https://github.com/Umio-Yasuno/amdgpu_top/assets/53935716/b5402110-c3b0-4e20-b756-fed30def1c9e)

**Comment by cyring** [comment]
Is this going to be an alternate UI or final one ?

Personally I much prefer your previous v0.1.7 look & feel; nVidia is an ugly style.

**Reply by Umio-Yasuno** [reply]
The roles are different for each mode.

- TUI
  - Selectable devices, single screen
- GUI
  - A device per window (process)
 - Simple TUI (SMI)
   - Multiple devices, single screen
